### DISCUSSION TITLE
Práctica HTML

### DISCUSSION THREAD

**Discussion by JoseEduardoVelazquezCN** [Launch-X-Latam/MisionFrontEnd#212] [Q&A]
Veo que muchos hablan ya de la práctica del módulo de HTML, pero en el repositorio no encuentro la práctica de HTML

Mi duda es no estoy en el grupo de frontend correcto o la práctica de HTML está en otro lado ¿? 

He visto una práctica de una pastelería pero nose si esa es la que realizaremos o será otra cuando actualicen el repositorio

**Comment by doguedogue** [comment]
Si estas asistiendo a las sesiones en vivo estas en el mismo canal, ahi informa que hacer el MC, es lo último al igual que en Teams.

También puedes navegar en los directorios del repositorio y veras sobre lo que hablan de la pasteleria: https://github.com/Launch-X-Latam/MisionFrontEnd/tree/main/02%20-%20HTML/practicas
